### ISSUE TITLE
Make fixed seed in sklearn

### ISSUE THREAD

**Issue by gryBox** [gryBox/learning_map#31]

**Comment by gryBox** [comment]
http://scikit-learn.org/stable/faq.html#how-do-i-set-a-random-state-for-an-entire-execution
